pkg/netpol/connlist/conns_formatter.go

```diff
@@ -86,21 +83,21 @@ const (
 	mapClose = "}"
 )
 
-// writeNsLabels returns a string representation of a potential peer with namespace labels
-func writeNsLabels(nsLabels map[string]string) string {
+// getRepresentativeNamespaceString returns a string representation of a potential peer with namespace labels
+func getRepresentativeNamespaceString(nsLabels map[string]string) string {
 	nsName, ok := nsLabels[common.K8sNsNameLabelKey]
 	if len(nsLabels) == 1 && ok {
 		return nsName
 	}
```

Comment:
is there a test that covers this in the output?

Reply:
yes, "tests/test_conn_to_all_pods_in_a_new_ns"